runner.go

```diff
@@ -75,10 +88,24 @@ func (s SplunkHealth) Flatten() map[string]Status {
 	return (Feature)(s).Flatten()
 }
 
+func expandEnv(s string) string {
+    if path, exists := os.LookupEnv("SPLUNK_HOME"); exists {
+        if filepath.IsAbs(path) {
+            if stat, err := os.Stat(path); err == nil && stat.IsDir() {
+                return os.ExpandEnv(s)
+            }
+        }
+    }
+    log.Fatal("SPLUNK_HOME must be a directory")
```

Comment:
Is this error all-encompassing for the paths that lead here?  Would we possibly get here if the directory just doesn't exist?

Reply:
Yes, the strings being expanded are constants that only reference ${SPLUNK_HOME}

This function is just to mitigate the risk of arbitrary command execution through that variable